File: api/WhatsGoodApi/WhatsGoodApi/Models/Weather/Wind/WindModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace WhatsGoodApi.Models.Weather.Wind
{
    public class WindModel : IWindModel
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }
        public decimal avg { get; set; }
        public decimal low { get; set; }
        public decimal high { get; set; }
        public string direction { get; set; }

    }
}

File: app/main/model/windModel.py
class windModel():

    high = 0.0
    low = 0.0
    avg = 0.0
    direction = "Default"

    def __init__(self, high, low, avg, direction):
        self.high = high
        self.low = low
        self.avg = avg
        self.direction = direction